web/assets/css/main.css

```diff
@@ -224,7 +224,6 @@ input[type='checkbox']:checked + label {
 }
 
 .dark .fc-scroller {
-    @apply scrollbarThin;
```

Comment:
I just wondered why @apply scrollbarThin is a circular dependency. There seem to be some false positives in this regard. Is this correct a warning or also a false positive?

Maybe https://github.com/tailwindlabs/tailwindcss/pull/6588 would change something about this warning (not sure)